packages/client/webapp/src/components/Sidebar.vue

```diff
@@ -0,0 +1,134 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import Button from '@/components/Button.vue';
+import SimulationPlan from '@carbon/icons-vue/es/data-player/32';
+import MachineLearningModel from '@carbon/icons-vue/es/machine-learning-model/32';
+import Dataset from '@carbon/icons-vue/es/table--split/32';
+import ProvenanceGraph from '@carbon/icons-vue/es/flow/32';
+import Search from '@carbon/icons-vue/es/search--locate/32';
+import User from '@carbon/icons-vue/es/user/32';
+
+const props = defineProps<{ sidebarPosition: String }>();
+const emit = defineEmits(['updateSidebarPosition']);
+
+const mode = ref('Simulation Plan');
+const collapsed = ref(false);
+
+function updateMode(modeOption: string) {
+	if (modeOption === mode.value) {
+		collapsed.value = !collapsed.value;
```

Comment:
Am I correct in thinking that this can be simplified to the following?

```suggestion
		collapsed.value = true;
```

Seems like if `modeOption === mode.value`, then collapsed must be false.

Reply:
No, it needs to be a toggle.

https://user-images.githubusercontent.com/636801/194637009-d66c2373-165c-4c40-9fe9-ddc97ff8cd92.mov

Reply:
I see. To me it's unintuitive that there would still be an active/selected tab even though the side panel is collapsed. Wouldn't it be more intuitive if mode.value is null or `''` by when the panel is collapsed?

Reply:
Yeah I see what you mean - vscode actually works like that. In this case though (from what I understand) what is displayed in the `<router-view/>` corresponds to what's selected in the sidebar. So it might make sense.

Reply:
Sorry @shawnyama but this meaning has changed right after our conversation...
So @bencodrington original opinion is now valid, and the one to go trough with.
The `<router-view/>` will not change based on the sidebar main buttons.

Reply:
Okay I'll change that then